bot/exts/evergreen/error_handler.py

```diff
@@ -42,8 +42,8 @@ def error_embed(message: str, title: Union[Iterable, str] = ERROR_REPLIES) -> Em
     @commands.Cog.listener()
     async def on_command_error(self, ctx: commands.Context, error: commands.CommandError) -> None:
         """Activates when a command opens an error."""
-        if hasattr(ctx.command, 'on_error'):
-            logging.debug("A command error occured but the command had it's own error handler.")
+        if hasattr(error, 'handled'):
```

Comment:
I'm tempted to say we should look at the value of handled here. Is there a situation where it could be false? I would opt for something like:
```suggestion
        if getattr(error, 'handled', False):
```

Reply:
I only reason I could think of settings the value to false is when you want the local error handler and the global error handler to handle the error:
```python
if isinstance(error, BadArgument):
    # Maybe user just wants to log some data in the local error handler
    error.handled = False  # default error handling by global error handler.
```
The only way it is kinda helping is readability by convaying the message that both the error handlers should be hit.
These situation are kinda rare and I thinking just not settings the attribute will  be easier.

Reply:
I think if there is a situation that makes sense we should try account for it, but I'm open to other opinions.

Those proposed situations don't sound like they are that unlikely, just not something we do now.

Reply:
alright 👍